### PR TITLE
Yarndings 20: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by emmamarichal**
16fd506: [gftools-packager] Yarndings 20: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

* Yarndings 20 Version 1.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/scfried/soft-type-yarndings at commit https://github.com/scfried/soft-type-yarndings/commit/200cc40a30ffbe590807cdfc8b8e17d0d63c43ff.

 e59fe9c: description and metadatas updated

### CODE REVIEW

**Comment by emmamarichal** [comment]
@vv-monsalve I updated the description with the sentence about the number.
For the metadatas (subsets), I suggest to let them like this, and try with this PR. If the font is well displayed, we can go ahead with the other PRs. I will let them in draft until we are happy with this one!

File: ofl/yarndings20/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p>The Soft Type Collection is designed for knitters to chart out their typographic projects. Yarndings is an ornamental typeface inspired by a mix of typographic ornamentation, traditional Fair Isle & Nordic knitting patterns, and dingbat fonts from the 1990's.</p>
+
+<p>Each typeface has a “Regular” and “Charted” version, and some include multiple scales so you can fit type on your knits, no matter the project's size</p>
```
Comment (vv-monsalve) [inline ofl/yarndings20/DESCRIPTION.en_us.html:3]: The link to the charted version is needed (following the same structure for all the Soft Type fonts)

**Review by vv-monsalve — CHANGES_REQUESTED** [review]

**Comment by vv-monsalve** [comment]
> For the metadatas (subsets), I suggest to let them like this, and try with this PR. If the font is well displayed, we can go ahead with the other PRs.

Okay, let's see hoe it perform in Dev.

**Comment by emmamarichal** [comment]
@vv-monsalve, done :)

**Review by vv-monsalve — APPROVED** [review]

**Comment by vv-monsalve** [comment]
Excellent! Now we just need to solve the tags issue (!)

**Pull request merged**